stories/conf.js

```diff
@@ -5,6 +5,9 @@ export const filterConfig = {
       'project',
       'study',
     ],
+    asTextAggFields: [
+      'consortium_id',
```

Comment:
this looks to break the storybook filters

Reply:
did you regenerate test data?

Reply:
yes i did, was a clean install locally

Reply:
weird... my storybook looks fine 🤔 
<img width="538" alt="Screenshot 2023-09-15 at 9 13 57 AM" src="https://github.com/uc-cdis/guppy/assets/2475897/ba69453e-506d-4026-9302-679b56e60c6a">

try re-run `sh ./generate_data.sh` ?

Reply:
figured it out, sorry will get back to testing, on ubuntu need to run the script with `./generate_data.sh` no sh as it does not run correctly :facepalm"